### PR TITLE
Adding Jasco Enbrighten ZB4102 plug and Philips LTA008 bulb

### PR DESCRIPTION
**Pull request by fifthecho**

### CODE REVIEW

File: devices/enbrighten.js [hunk 1]
```diff
@@ -88,4 +88,16 @@ module.exports = [
             await reporting.onOff(endpoint);
         },
     },
+    {
+        zigbeeModel: ['43094'],
+        model: '43094',
+        vendor: 'Jasco Products',
```
Comment (Koenkk) [inline devices/enbrighten.js:94]: Please use `Enbrighten` here or move it to a new file (`jasco.js`)

Comment (fifthecho) [reply devices/enbrighten.js:94]: Dumb question (as it's my first Zigbee code) - the device advertises itself as "Jasco Products" - would this be a problem? Or is this just an identifier used by Zigbee2MQTT?

Comment (Koenkk) [reply devices/enbrighten.js:94]: It is just a cosmetic thing, seems it should use `Enbrighten ` (https://www.youtube.com/watch?v=e4TMSGBuLLw&ab_channel=JascoProductsCompany)

Comment (fifthecho) [reply devices/enbrighten.js]: Cool. Done and pushed.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**